gamestonk_terminal/cryptocurrency/defi/coindix_model.py

```diff
@@ -0,0 +1,130 @@
+"""Coindix model"""
+__docformat__ = "numpy"
+
+from typing import Optional
+
+import requests
+import pandas as pd
+
+VAULTS_FILTERS = ["name", "chain", "protocol", "apy", "tvl", "risk"]
```

Comment:
Since the API also provides this I think we should include a link to the vault that will show if --link is passed